package/network/services/lldpd/files/lldpd.init

```diff
@@ -90,10 +102,12 @@ start_service() {
 	procd_open_instance
 	procd_set_param command /usr/sbin/lldpd -d
 
-	[ $enable_cdp -gt 0 ] && procd_append_param command '-c'
-	[ $enable_fdp -gt 0 ] && procd_append_param command '-f'
-	[ $enable_sonmp -gt 0 ] && procd_append_param command '-s'
-	[ $enable_edp -gt 0 ] && procd_append_param command '-e'
+	[ $enable_cdp -gt 0 ] && procd_append_param command "-$(printf 'c%.0s' $(seq 1 $enable_cdp))"
+	[ $enable_fdp -gt 0 ] && procd_append_param command "-$(printf 'f%.0s' $(seq 1 $enable_fdp))"
+	[ $enable_sonmp -gt 0 ] && procd_append_param command "-$(printf 's%.0s' $(seq 1 $enable_sonmp))"
+	[ $enable_edp -gt 0 ] && procd_append_param command "-$(printf 'e%.0s' $(seq 1 $enable_edp))"
```

Comment:
These 4 options are all boolean, they are only 0 or 1 `"-$(printf 'e%.0s' $(seq 1 $enable_edp))"` does not change it.

It would also be better if you do this change in a separate commit.

Reply:
Mhh, I can't find the code section. It seems that a better solution has already been found. (Especially with CDP, as there are quite a few possibilities there).